app/models/npq_lead_provider.rb

```diff
@@ -4,4 +4,8 @@ class NPQLeadProvider < ApplicationRecord
   belongs_to :cpd_lead_provider, optional: true
 
   has_many :npq_profiles, class_name: "NPQValidationData"
+
+  def participants
+    User.where(id: npq_profiles.map(&:user_id))
+  end
```

Comment:
Can we get it by `has_many :npq_participants, through: :npq_profiles, source: :user`?

Reply:
☝️ or at the very least use `pluck` instead of `map`, which loads the full object.

Reply:
Done